src/main/kotlin/org/rust/ide/inspections/fixes/InitializeWithDefaultValueFix.kt

```diff
@@ -47,6 +49,14 @@ class InitializeWithDefaultValueFix(element: RsElement) : LocalQuickFixAndIntent
         }
     }
 
+    private fun getVisibleBindings(place: RsElement) : HashSet<RsPatBinding> {
```

Comment:
This is duplicated in AddStructFieldsFix. Where is a good place to put it to avoid the duplication?